frontend-react/src/components/annotate.js

```diff
@@ -2,70 +2,72 @@ import React, { Component } from 'react';
 import ReactImageAnnotate from "react-image-annotate";
 import '../style/annotate.css';
 
-// TODO: this needs to be cleaned up
 class Annotate extends Component {
-    constructor(props) {
-        super(props);
-        this.state = {
-            file: '',
-            imagePreviewUrl: ''
-        };
-        this._handleImageChange = this._handleImageChange.bind(this);
-        this._handleSubmit = this._handleSubmit.bind(this);
-      }
-
-      _handleSubmit(e) {
-          e.preventDefault();
-          // TODO: do something with -> this.state.file
-      }
 
-      _handleImageChange(e) {
-          e.preventDefault();
+    state = {
+        data: [],
+        uploaded: false
+    }
 
-          let reader = new FileReader();
-          let file = e.target.files[0];
+  fileSelectedHandler = event => {
+    var images = [];
+    let files = event.target.files;
+    var img = new Image();
+    for (let i = 0; i < files.length; i++) {
+        img.src = URL.createObjectURL(files[i]);
+        images.push({src: img.src, name: files[i].name});  
+    }
 
-          reader.onloadend = () => {
-              this.setState({
-                  file: file,
-                  imagePreviewUrl: reader.result
-              });
-          };
+    this.setState({data: this.state.data.concat(images)});
+    this.setState({uploaded: true});
+  }    
 
-          reader.readAsDataURL(file);
-      }
-    render() {
-        let {imagePreviewUrl} = this.state;
-        let $imagePreview = null;
-        if (imagePreviewUrl) {
-            $imagePreview = (<img src={imagePreviewUrl} />);
-        }
-
-        return (
-            <div className="border">
-                <form className="selectForm" onSubmit={this._handleSubmit}>
-                    <input type="file" id="file" className="fileInputButton" onChange={this._handleImageChange} />
+      render() {
+        let { data } = this.state;
+        let { uploaded } = this.state;
+        if(uploaded){
+          return (
+              <div className="border">
+                  <form className="selectForm" onSubmit={this._handleSubmit}>
+                    <input type="file" id="file" multiple={true} accept="image/*" className="fileInputButton" onChange={this.fileSelectedHandler} />
                     <label htmlFor="file" className="fileLabel">
-                        Select Image
+                      Add Images
                     </label>
-                </form>
+                  </form>
                 <div className="imageDisplay">
+                  <ReactImageAnnotate
+                    images= {data}
+                    key = {data}
+                    regionClsList={[]}
```

Comment:
I am no longer able to add a label to annotations. I think we may need to leave some default annotations or the empty string `regionClsList={[""]}`